src/main/java/de/adesso/anki/battle/world/bodies/Rocket.java

```diff
@@ -1,24 +1,45 @@
 package de.adesso.anki.battle.world.bodies;
 
+import java.util.Iterator;
 import java.util.List;
+import java.util.Timer;

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.Timer'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)